examples/games/breakout.rs

```diff
@@ -348,7 +362,10 @@ fn apply_velocity(mut query: Query<(&mut Transform, &Velocity)>) {
     }
 }
 
-fn update_scoreboard(scoreboard: Res<Scoreboard>, mut query: Query<&mut Text>) {
+fn update_scoreboard(
+    scoreboard: Res<Scoreboard>,
+    mut query: Query<&mut Text, Without<SteppingUi>>,
```

Comment:
The logic here should be flipped: the scoreboard text should really have a marker component.

Reply:
Haha, I tried to add the `Scoreboard` component and discovered it already existed as a resource.  I'll make a `ScoreboardUi` instead.